frontend/views/containers/group-settings/GroupLeaveModal.vue

```diff
@@ -126,7 +126,7 @@ export default ({
       confirmation: {
         [L('This field is required')]: required,
         [L('Does not match')]: function (value) {
-          return normalizeString(value) === normalizeString(this.code)
+          return normalizeString(value || '') === normalizeString(this.code)
```

Comment:
Can this possibly not be a string? Even then, maybe `??` is better than `||`, since it avoids having a fallback for `''` (i.e., if value is `''`. This is a minor revision though.